### PR TITLE
Modify constructors

### PR DESCRIPTION
**Pull request by Mohammadreza-73**
Use constructor property promotion.

### CODE REVIEW

File: src/Dto/SentOtpDto.php [hunk 1]
```diff
@@ -6,18 +6,11 @@
...
-    {
-        $this->code = $code;
-        $this->waitingTime = $waitingTime;
-        $this->trackingCode = $trackingCode;
-    }
+    public function __construct(
+        public int $code,
+        private int $waitingTime,
+        public string $trackingCode
+    ) {}
```
Comment (salehhashemi1992) [inline src/Dto/SentOtpDto.php:13]:
```suggestion
    ) {
+    }
```

File: src/Events/OtpPrepared.php [hunk 2]
```diff
@@ -10,18 +10,13 @@ class OtpPrepared
...
      */
-    public function __construct(string $mobile, string $code)
-    {
-        $this->mobile = $mobile;
-        $this->code = $code;
-    }
+    public function __construct(
+        public string $mobile,
+        public string $code
+    ) {}
```
Comment (salehhashemi1992) [inline src/Events/OtpPrepared.php:21]:
```suggestion
    ) {
+    }
```

File: src/Dto/OtpDto.php [hunk 3]
```diff
@@ -6,15 +6,10 @@
...
-
-    public function __construct(int $code, string $trackingCode)
-    {
-        $this->code = $code;
-        $this->trackingCode = $trackingCode;
-    }
+    public function __construct(
+        public int $code,
+        public string $trackingCode
+    ) {}
```
Comment (salehhashemi1992) [inline src/Dto/OtpDto.php:12]:
```suggestion
    ) {
+    }
```

**Comment by salehhashemi1992** [comment]
Thank you. Please fix the ci errors.

**Comment by salehhashemi1992** [comment]
@Mohammadreza-73

**Comment by Mohammadreza-73** [comment]
@salehhashemi1992 
I think you overused the Github actions.

**Comment by salehhashemi1992** [comment]
@Mohammadreza-73 You are not familier with styleci? if you check every Laravel package you can find it in the ci flow! there are rules to follow when you wanna write a Laravel package.

**Pull request closed without merging**